ursa_core/.gitignore

```diff
@@ -0,0 +1 @@
+Cargo.lock
```

Comment:
Shared objects are a murky area. I do think that given that Ursa is a primarily going to be used as an rlib, this is the right call.